src/Sylius/Bundle/UiBundle/Resources/views/Grid/_default.html.twig

```diff
@@ -34,7 +34,7 @@
     </div>
 {% endif %}
 
-{% if definition.actionGroups.bulk is defined %}
+{% if data|length > 0 and definition.actionGroups.bulk is defined %}
```

Comment:
Do we need this `data|length` in here?

Reply:
I guess yes, because we shouldn't display block with bulk buttons, when the grid is empty